src/components/CreateMatch/CreateMatch.js

```diff
@@ -1,40 +1,135 @@
-import React from 'react';
-import './CreateMatch.scss';
+import React,{ Component } from 'react';
 import create from '../../images/create-icon.png';
 import soccerField from '../../images/soccer-field.jpg';
 import players from '../../images/player.png';
 import shirts from '../../images/shirt-options.png';
 import calendar from '../../images/date.png';
 import field from '../../images/field-option.png';
 import configure from '../../images/configure.png';
-import deleteTeam from '../../images/delete-team.png';
-
-const CreateMatch = () => {
-    return (
-        <div className="match">
-            <span className="match-header">
-                <span className="match-header__title">Crear Partido</span>
-                <span className="match-header__icon"><img className="match-header__icon-img" src={create} alt="crear partido" /></span>
+import close from '../../images/delete-team.png';
+import shield from '../../images/shield.png';
+import Team from './Team/Team';
+import CreateTeam from './Team/CreateTeam/CreateTeam';
+import check from '../../images/check-icon.png';
+import deleteIcon from '../../images/delete-icon.png';
+import editIcon from '../../images/edit-icon.png';
+import ballPlay from '../../images/ball-play.png';
+import captain from '../../images/captain.png';
+import Users from '../../Library/Users';
+import './CreateMatch.scss';
+
+class CreateMatch extends Component { 
+    constructor () {
+        super();
+        this.state = {
+            addPlayer: false,
+            createTeam: false,
+            name: '',
+            nickname: '',
+            players: '',
+            containerInput: true,
+            namePlayer: '',
+            numberPlayer: '',
+            editPlayer: false,
+        }
+    }
+
+    handleOnChange = e => {
+        const { target: { value, name } } = e
+        this.setState({        
+            [name]: value      
+        }); 
+    }
+
+    toggleAddPlayer = () => {
+        this.setState({
+            addPlayer: !this.state.addPlayer,
+        })
+    };
+
+    showCreateTeam = () => {
+        this.setState ({
+            createTeam: !this.state.createTeam,
+        })
+    };
+
+    hideInfoInput = () => {
+        const { name,  nickname, players, } = this.state;
+        if( name !== '' && nickname !== '' && players !== ''){
+            this.setState({
+                containerInput:!this.state.containerInput,
+            });
+        }
+    };
+
+    showPlayersList = () => {
+        const database = Users;
+        return database.map((singlePLayer) => <span className="list-player" key={singlePLayer.name}>
```

Comment:
It is not necessary to create a new variable please use Users instead of the new variable database. (Delete line 66 and use Users instead of database on line 67